### PR TITLE
Add file extension "mjs" to list of translatable files

### LINKED ISSUE #354: l10n: Add mjs to list of javascript filename extensions
App gestion started to use javascript files with extension "mjs".

mjs is a extension for module files.

Docs:
* https://v8.dev/features/modules#mjs
* https://v8.dev/features/modules#mjs

See https://github.com/baimard/gestion/issues/146 as issue reference.

### PR DESCRIPTION
**Pull request by rakekniven**
Fixes #354 

Should fix https://github.com/baimard/gestion/issues/146 as well.

Signed-off-by: rakekniven <2069590+rakekniven@users.noreply.github.com>

### CODE REVIEW

File: translations/translationtool/src/translationtool.php [hunk 1]
```diff
@@ -67,7 +67,7 @@ public function createPotFile() {
 		$this->createFakeFileForVueFiles();
 		$this->createFakeFileForLocale();
 		$translatableFiles = $this->findTranslatableFiles(
-			['.php', '.js', '.jsx', '.html', '.ts', '.tsx'],
+			['.php', '.js', '.jsx', 'mjs', '.html', '.ts', '.tsx'],
```
Comment (nickvergessen) [inline translations/translationtool/src/translationtool.php:70]:
```suggestion
			['.php', '.js', '.jsx', '.mjs', '.html', '.ts', '.tsx'],
```

**Review by nickvergessen — APPROVED** [review]

**Comment by rakekniven** [comment]
Hello @nickvergessen ,

does this PR needs to be deployed somewhere after it has been merged?
I cannot see source strings appearing (see linked issue from initial post) at Transifex.

**Comment by nickvergessen** [comment]
Yes i guess so.
Ill have a look this week. I also have an error from logreader coming in daily as a reminder

**Comment by rakekniven** [comment]
No new strings from mjs as source files this morning.

**Comment by baimard** [comment]
@nickvergessen is it possible to reopen this issue ?

**Comment by rakekniven** [comment]
@nickvergessen This PR has been closed. But the strings from MJS files are still missing.
As there is no issue for this I am kindly asking for help to solve this.

**Pull request merged**